README.md

```diff
@@ -31,18 +31,31 @@ On macOS: `brew install cmake`
 
 ## Build
 
+To unit test build run:
+
+```shell
+mkdir build && cd build
+cmake -DCMAKE_BUILD_TYPE=Debug .
```

Comment:
This command will not work since there is no CMake file in the `build` folder.

Reply:
Rewrote the command in readme